bedrock/firefox/views.py

```diff
@@ -220,9 +220,13 @@ def send_to_device_ajax(request):
 
                 try:
                     basket.request('post', 'subscribe_sms', data=data)
-                except basket.BasketException:
-                    return HttpResponseJSON({'success': False, 'errors': ['system']},
-                                            status=400)
+                except basket.BasketException as e:
+                    if e.desc == 'mobile_number is invalid':
```

Comment:
To clarify, this error will come directly from basket, correct? Could/should we do any validation (e.g. make sure length is sufficient) on our end first to avoid the round trip?

Reply:
It's really tough to do that now. I can maybe find out what the max length of any phone number in the world is, but there is a max on the form field that will prevent any ridiculous entries.

Reply:
I was more curious about numbers being too short - like someone accidentally hit enter after putting in only 3 digits or something.

Reply:
Hmm... perhaps... I'll look into it.